client/components/TestPlanVersionsPage/index.jsx

```diff
@@ -267,11 +274,70 @@ const TestPlanVersionsPage = () => {
                             ))}
                         </tbody>
                     </ThemeTable>
+                    <PageSpacer />
                 </>
             )}
-            {/* TODO: enable table when issues are present */}
-            {/* <ThemeTableHeader>GitHub Issues</ThemeTableHeader>
-            <ThemeTableUnavailable>No GitHub Issues</ThemeTableUnavailable> */}
+
+            <ThemeTableHeader>GitHub Issues</ThemeTableHeader>
+            {!issues.length ? (
+                <ThemeTableUnavailable>No GitHub Issues</ThemeTableUnavailable>
+            ) : (
+                <ThemeTable bordered responsive>
+                    <thead>
+                        <tr>
+                            <th>Author</th>
+                            <th>Issue</th>
```

Comment:
I wonder if having an additional column dedicated to the AT the issue is filed against, would be helpful in quickly parsing the content of the issues table

Reply:
Nice idea, added.